lib/plausible_web/controllers/api/stats_controller.ex

```diff
@@ -16,21 +16,42 @@ defmodule PlausibleWeb.Api.StatsController do
         query
       end
 
-    timeseries = Task.async(fn -> Stats.timeseries(site, timeseries_query, ["visitors"]) end)
+    timeseries = Task.async(fn -> Stats.timeseries(site, timeseries_query, [:visitors]) end)
     {top_stats, sample_percent} = fetch_top_stats(site, query)
 
     timeseries_result = Task.await(timeseries)
-    plot = Enum.map(timeseries_result, fn row -> row["visitors"] end)
+    plot = Enum.map(timeseries_result, fn row -> row[:visitors] end)
     labels = Enum.map(timeseries_result, fn row -> row["date"] end)
     present_index = present_index_for(site, query, labels)
 
+    {plot, with_imported, source} =
+      if query.with_imported && site.imported_source do
+        # Showing imported data.
+        plot =
+          Stats.Imported.timeseries(site, timeseries_query)
+          |> Enum.zip_with(plot, &(&1 + &2))
+
+        {plot, true, site.imported_source}
+      else
+        if Map.get(params, "filters", "{}") != "{}" do
```

Comment:
This line was a bit weird to read. How about `if Map.has_key?("filters") do`?

Reply:
Sometimes the key is there but the value is `"{}"` and that should also pass. But perhaps this is also better?:
```suggestion
        if not Map.has_key?("filters") or params["filters"] == "{}" do 
```

Reply:
Ah I see, this is the raw filters parameter.

The filters are parsed a few lines above using `Query.from(params)`. You could use the result from that on this line to make it cleaner:

```elixir
if Enum.any?(query.filters) do
   # ...
end
```

Reply:
Ah yes that's much better